pkgs/applications/window-managers/wmii-hg/default.nix

```diff
@@ -29,7 +30,7 @@ stdenv.mkDerivation rec {
     EOF
   '';
 
-  buildInputs = [ pkgconfig libixp_hg txt2tags dash python
+  buildInputs = [ pkgconfig libixp_hg txt2tags dash python which
```

Comment:
It makes no real difference, but this is strictly unnecessary if all you're doing is writing a reference to the store path into the shell script.